Docs for customizing drill-through

mazameli opened the pull request:
Feels a bit verbose, so gonna do another pass to break it up a bit and see if I can cut some verbiage.

mazameli committed:
docs for customizing drill-through

mazameli committed:
fix img name

mazameli committed:
flesh out instructions for linking to dashboards

mazameli committed:
small formatting edits

mazameli committed:
move things to enterprise guide and add some images

mazameli committed:
enterprise guide toc init

kdoh commented on docs/enterprise-guide/customizing-drill-through.md:
```diff
@@ -0,0 +1,76 @@
+
+**Customizing link text**
+
+Additionally, you can optionally customize what text should be displayed in each cell of this column. You can also use the double braces variable syntax here. The way we've used this in the example above is to refer to the value of our `Product ID` column in a sentence that makes it clearer what will happen when the cell is clicked. So `Look up {{PRODUCT_ID}} in our catalog` gets turned into `Loo ...
```
`Additionally, you can optionally` stood out as a bit awkward to me.

kdoh commented on docs/enterprise-guide/customizing-drill-through.md:
```diff
@@ -0,0 +1,76 @@
+
+Again, in our example our table visualization is in a dashboard, so we’ll start by entering edit mode for our dashboard and then clicking the settings gear button on our table. If you were doing this on a standalone question, you’d click the gear button next to the visualization dropdown.
+
+From the `Columns` tab I'll find my `Product ID` column and click the gear next to it.
```
The switch from `we` to `I` stood out here and the other instance below.

mazameli committed:
move enterprise feature docs and images to new home

mazameli committed:
reel good gramer

mazameli committed:
remove asterisks

mazameli committed:
cleanup

mazameli commented:
This PR now additionally moves docs for enterprise features into a new enterprise edition guide (and out of the Admin Guide).

For now, instead of deleting the old doc pages that had been in the Admin Guide, I removed them from that Admin Guide TOC, fixed up all the "next" links for other docs that previously linked to them, and put a note in the old pages that they've been moved to a new location.

The new enterprise edition guide doesn't yet have "next" links in the footers. Personally I'd rather move to having a fixed sidebar TOC on doc pages for the current guide you're in, but I'm not sure how to do that.

mazameli committed:
add Next links

mazameli commented:
In an effort to get this merged, I've gone ahead and just added Next links to the footers of the enterprise doc pages. I'd like to tackle adding a persistent sidebar TOC to each of our Guides in a separate PR.

Can I ask you to review please, @salsakran?

salsakran reviewed (approved):
the structure works for me

kdoh commented on docs/users-guide/start.md:
```diff
@@ -17,5 +17,6 @@
 *   [Creating SQL Templates](13-sql-parameters.md)
 *   [Viewing X-ray reports](14-x-rays.md)
 *   [Getting alerts](15-alerts.md)
+*   [Customizing drill-through](16-customizing.drill-through.md)
```
Should we still be linking to this in the users guide?

mazameli replied:
No, good catch!

mazameli committed:
remove custom drill through from users guide
